project/README.rst

```diff
@@ -82,5 +82,5 @@ Time Table
 Project List by Account Name
 ++++++++++++++++++++++++++++
 
-#. `github username <https://github.com/yungyuc>`__ for
-   `project name <README.rst>`__: project repository URL
+#. `minchuncho <https://github.com/minchuncho>`__ for
```

Comment:
Could you help resolve the merge conflict that I caused by a previous merge?  I think rebasing from the upstream would just do it.

It's not your fault but your help will save me time.

Reply:
Sure do. If I have done it incorrectly, plz let me know, thank you. @yungyuc

Reply:
This is good.  Thanks.